src/annotator/anchoring/pdf.js

```diff
@@ -568,17 +568,14 @@ export async function anchor(root, selectors) {
 }
 
 /**
- * Convert a DOM Range object into a set of selectors.
+ * Prepare a DOM range for generating selectors and find the containing text layer.
  *
- * Converts a DOM `Range` object into a `[position, quote]` tuple of selectors
- * which can be saved with an annotation and later passed to `anchor` to
- * convert the selectors back to a `Range`.
+ * Throws if the range cannot be annotated.
```

Comment:
I personally feel it is more obvious to annotate the function than to parse the comments.

```suggestion
 * @throws {Error} if the range cannot be annotated.
```

Reply:
Done. I think we can probably omit the `Error` type and just assume it if there is no custom error. One caveat also with types is that TypeScript doesn't do anything with them at present. It cannot for example verify that a function really throws what it claims to.